packages/relay/test/report.test.ts

```diff
@@ -579,5 +580,146 @@ describe('POST /api/report', function () {
                 expect(res).to.have.status(400)
                 expect(res.body.error).to.be.equal('Report voting has ended')
             })
+        it('should settle report and update object status', async function () {
```

Comment:
all settle report tests are under `'should fail if vote on the report whose status is not VOTING'` test, should be moved out.